doc/source/admin/galaxy_options.rst

```diff
@@ -477,6 +477,20 @@
 :Type: str
 
 
+~~~~~~~~~~~~~~~~~~~~~
+``watch_core_config``
+~~~~~~~~~~~~~~~~~~~~~
+
+:Description:
+    Set to true to enable monitoring of a subset of options in the
```

Comment:
Would you be opposed to `Monitor of a subset of options in the ...` ?

Reply:
Not at all!

Reply:
Also changing the other 3 occurrences of the same wording (all in config_schema.yml)